hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/diskbalancer/DiskBalancerService.java

```diff
@@ -392,6 +392,9 @@ private class DiskBalancerTask implements BackgroundTask {
 
     @Override
     public BackgroundTaskResult call() {
+      //start time
+      long startTime = Time.monotonicNow();
+      boolean moveSucceeded = false;
```

Comment:
Nit: I prefer to have `moveSucceeded` to be `true` and only set to false in `exception` so that the main body is try catch body is cleaner.